### PR TITLE
refactor(ui): redesign org profile tab

### LINKED ISSUE #15964: Display org ID and name in org profile page

### PR DESCRIPTION
**Pull request by alexpaxton**
Closes #15964 

Display organization name and id in the "Org Profile" settings tab

Screen Shot 2019-11-18 at 2 32 15 PM

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/organizations/containers/OrgProfilePage.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ class OrgProfilePage extends Component<StateProps> {
             <Grid>
               <Grid.Row>
                 <Grid.Column widthXS={Columns.Twelve} widthSM={Columns.Six}>
-                  <OrgProfileTab />
+                  <OrgProfileTab orgID={org.id} orgName={org.name} />
```
Comment (drdelambre) [inline ui/src/organizations/containers/OrgProfilePage.tsx]: why isn't it just grabbing it's own org info from the store?

Comment (alexpaxton) [reply ui/src/organizations/containers/OrgProfilePage.tsx]: That's a good question. Long live discreet components!!

**Comment by alexpaxton** [comment]
@timhallinflux mentioned we should have a show/hide feature for this type of sensitive information (org id and tokens). I am going to put this on the backburner until I have a more standardized component to handle that interaction

File: ui/src/organizations/components/OrgProfileTab.tsx [hunk 2]
```diff
@@ -1,84 +1,102 @@
...
+}
 
-  private handleShowEditOverlay = () => {
-    const {
-      params: {orgID},
-      router,
-    } = this.props
+type Props = WithRouterProps & OwnProps
 
+const OrgProfileTab: FunctionComponent<Props> = ({orgID, orgName, router}) => {
```
Comment (121watts) [inline ui/src/organizations/components/OrgProfileTab.tsx]:
```suggestion
const OrgProfileTab: FC<Props> = ({orgID, orgName, router}) => {
```

File: ui/src/organizations/components/OrgProfileTab.tsx [hunk 3]
```diff
@@ -1,84 +1,102 @@
 // Libraries
-import React, {PureComponent} from 'react'
+import React, {FunctionComponent} from 'react'
```
Comment (121watts) [inline ui/src/organizations/components/OrgProfileTab.tsx]:
```suggestion
import React, {FC} from 'react'
```

**Comment by russorat** [comment]
@alexpaxton we should fix the conflicts and merge this. It would be very helpful for people using the API or client libraries.

**Comment by wbaker85** [comment]
The UI has changed quite a bit since this PR was opened, and now does indeed show the org name and ID in the "about" page, so I'm closing this!

**Pull request closed without merging**